tools/devtool

```diff
@@ -2166,11 +2172,11 @@ cmd_build_rootfs() {
 
     truncate -s "$SIZE" "$img_file"
     mkfs.ext4 -F "$img_file"
-    docker run --env ROOTFS_NAME=$ROOTFS_NAME --env PARTUUID=$PARTUUID --privileged --rm -i -v "$FC_ROOT_DIR:/firecracker"  "$FROM_CTR" bash -s <<'EOF'
-rootfs_dir=/firecracker/build/rootfs
+    docker run --env ROOTFS_DIR="$ROOTFS_DIR" --env MNT_DIR="$MNT_DIR" --env ROOTFS_NAME=$ROOTFS_NAME --env PARTUUID=$PARTUUID --privileged --rm -i -v "$FC_ROOT_DIR:/firecracker"  "$FROM_CTR" bash -s <<'EOF'
+rootfs_dir="$ROOTFS_DIR"
 img_file="$rootfs_dir/$ROOTFS_NAME"
```

Comment:
I don't see the need for an additional local variable here, you could just use `$ROOTFS_DIR`:

```suggestion
img_file="$ROOTFS_DIR/$ROOTFS_NAME"
```